### PR TITLE
After two years in hell...

### PR DESCRIPTION
**Pull request by batflarrow**
Obviously everybody's doing this for the t-shirt, but for me its just and added benefit.The workshop taught me something entirely new, and it was a fun experience. Please do read my file in _posts, it might amuse you.

Thanks;

### CODE REVIEW

File: _posts/2019-10-05-amit.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: "Amitvikram Dwivedi"
+github_id: "batflarrow"
+---
+
+Heya everyone, myself batflarrorw, the combination of Batman Flash and Arrow.After two years of hell in jee coaching, i finally reached here.I thought I'll live "Majegi ki Zindagi", but frankly its the same hell:(.We werent allowed to use mobiles and basically internet before, but now even though we have devices the internet is pretty much shit.Sala har mahine ek exam daaldete hain, kya pure saal ham padhte hi rhe kya. These kind of workshops and fest are the only thing that separates us from the shithole of a life we are living.I would really like to thank our seniors for organising such events and bringing us away from hell.
```
Comment (rossoskull) [inline _posts/2019-10-05-amit.md]: I really appreciate the comment 😄  But make sure it is only 1-2 lines, 10-20 words. And also remove the other lines which are not needed.

**Review by rossoskull — COMMENTED** [review]
Please make the changes suggested so that I can review and merge it.

**Comment by rossoskull** [comment]
Close this PR and reopen a new one with only a single commit.

**Comment by anshumanv** [comment]
> Obviously everybody's doing this for the t-shirt, but for me its just and added benefit.

Yes, the key takeaway is the experience and not the incentives, great! ✨

**Pull request closed without merging**